### ISSUE TITLE
[4.0] Delete and disable action icons are duplicated

### ISSUE THREAD

**Issue by Kostelano** [joomla/joomla-cms#33432]
### Steps to reproduce the issue
Go to System - Update sites, select any site and click the Actions button. I would prefer that the icons are not duplicated. Probably the trash icon would be more appropriate here. 

![Screenshot_1](https://user-images.githubusercontent.com/8440661/116733082-53531400-a9f4-11eb-845a-b0b5bcc8d919.png)

**Comment by YatharthVyas** [comment]
I agree that both the icons appear visually similar but these icons have different classes:

- `icon-unpublish` for disable

![image](https://user-images.githubusercontent.com/53610833/116774630-93a9a500-aa7b-11eb-9ce3-a5cd78cfb7b4.png)

- `icon-delete` for delete

![image](https://user-images.githubusercontent.com/53610833/116774638-9f956700-aa7b-11eb-8904-8e515b5e8387.png)

If we were to change the delete icon to `icon-trash`, It might give the users a false feeling of Trashing the item (similar to articles) and that hope that they can recover this trashed item later which is not the case with delete. In my opinion, this would not be in the best interest of preserving the consistency of icons across different modules.

![image](https://user-images.githubusercontent.com/53610833/116774751-4a0d8a00-aa7c-11eb-9827-fe1b1c9d2418.png)

**Comment by PhilETaylor** [comment]
In joomla trash and delete should and do mean different things

Trash = move to trash status which can be changed back to another status

Delete = gone forever, deleted

**Comment by drmenzelit** [comment]
`icon-unpublished` and `icon-delete` (and probably others too) are both mapped to `$fa-var-times` in `\system\scss\_icomoon.scss`. A specific class `icon-disable` will be helpful, for example as `fas fa-ban`

**Comment by Kostelano** [comment]
PR https://github.com/joomla/joomla-cms/pull/33515

**Comment by Hackwar** [comment]
This has since been fixed in the code base. Thanks for reporting.
